<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox1</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox2</string>
    </property>
    <widget class="QPushButton" name="pb2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Count sum</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="le5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>patronymic</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>surname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>phoneNumber</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>131</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>averageGrade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Create an Object</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
